src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -259,10 +260,17 @@ public function shouldReport(Throwable $e)
      */
     protected function shouldntReport(Throwable $e)
     {
-        $dontReport = array_merge($this->dontReport, $this->internalDontReport);
+        $dontReport = collect($this->internalDontReport)
+            ->mapWithKeys(function ($type) {
+                return [$type => true];
+            })
+            ->merge($this->dontReport);
```

Comment:
> I'm just saying currently your code assumes $dontReport is already formatted with the class names as the keys.

Ah I see - yes. So here it needs to ensure `$dontReport` is an associative array before merging?